pkg/cmd/logs.go

```diff
@@ -156,17 +156,22 @@ func runCommand(args []string) {
 	case "cluster-autoscaler":
 		logsClusterAutoscaler()
 	case "tf":
-		if len(args) == 1 {
-			logsTf()
+		if len(args) == 1 || len(args) < 3 {
+			logsTfHelp()
 			break
 		}
+
+		var prefixName string = (args[02])
```

Comment:
can we also check if shoot name exists and please use `args[2]`?

Reply:
for my understanding, so far the logic for `logs tf` will fetch all `podName` and `podNamespace`  from a seed node, then match `infra.tf` from List output to found the pod name.   
It takes about at least 11M List output data  when I testing, depends on the total of shoots host in the seed node. 
To check shoot name whether exists, it maybe double the times for the result output. search shoot name from List output ->exist -> search `infra.tf` from List output to found the pod name.  Let me know any concerns. thanks 


[https://github.com/gardener/gardenctl/blob/f68840200614710714ba6df3cdc7626ea4be7ecc/pkg/cmd/logs.go#L506-L522](https://github.com/gardener/gardenctl/blob/f68840200614710714ba6df3cdc7626ea4be7ecc/pkg/cmd/logs.go#L506-L522)